RoboSharp/RoboCommand.cs

```diff
@@ -277,12 +279,18 @@ void process_ErrorDataReceived(object sender, DataReceivedEventArgs e)
                 OnCommandError(this, new ErrorEventArgs(e.Data));
             }
         }
-
+        
+	    void Process_Exited(object sender, System.EventArgs e)
+	    {
+		    hasExited = true;
+    	}
```

Comment:
You are mixing tabs and spaces in this block, seems the rest of the file sticks to spaces.